scripts/update-github-actions.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# This script calls the dependabot CLI (https://github.com/dependabot/cli)
+# to determine updates to GitHub Action dependencies in the local repository.
+# It then also applies the updates and outputs the results to standard output.
+
+set -euo pipefail
+
+REPO_ROOT=$1
+
+echo -e "<details><summary>GitHub Action updates</summary>\n\n"
+
+# Each dependabot update call tries to update all dependencies,
+# but the resulting files are output individually for each (with the intention of creating a PR for each).
+# We want to have all changes together though, so we just repeatedly take the first one
+# until there's none anymore (-e jq flag)
```

Comment:
That's not quite what `-e` does by my reading of [the manual](https://jqlang.github.io/jq/manual/#invoking-jq):

> > `--exit-status / -e:`
>
> Sets the exit status of jq to 0 if the last output value was neither false nor null, 1 if the last output value was either false or null, or 4 if no valid result was ever produced. Normally jq exits with 2 if there was any usage problem or system error, 3 if there was a jq program compile error, or 0 if the jq program ran.

Reply:
Good eye, I was kind of lazy with the wording there, will improve that!

Reply:
[(diff)](https://github.com/NixOS/nixpkgs-check-by-name/compare/08e962950f372595027bcbf898a012a97b62d96b..815a4ecd3031e43751baf65c0b2722a6346c6486#diff-a6887f32bd809f67897cc1dd3164202dfe5a6d4933b6f736b3ae864e00a70893R17-R32)